modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -1209,6 +1209,51 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViweOnlyCorrectAllowedQuantitiesAreSelectableInProductCard {
```

Comment:
CanViewOnlyCorrectAllowedQuantitiesAreSelectableForProduct

Please combine these 2 tests and make an assertion first on the product card, then on the product details page. It's a repetition and can be worked under one test. Thanks!